voluncheer/templates/opportunityboard/postanopportunity.html

```diff
@@ -112,6 +112,23 @@ <h2>OPPORTUNITY DETAILS</h2>
                     });
                   });
                 </script>
+                <script>
```

Comment:
Can we just use a timedelta instead? Also, what if you made a separate .js file  to contain this for readability (see Wayne's room.js in the chatroom template).

Reply:
There's no build-in time delta in javascript. And sure I can move all script to a separate file. Thanks!